setup.py

```diff
@@ -21,7 +21,11 @@
             "pytest==2.9.1",
         ]
     },
-    dependency_links=[],
+    entry_points={
```

Comment:
I don't think that is possible via setuptools entrypoints. Any fixtures defined in a project will shadow these anyway. I think the only risk of a clash is a similarly named fixture in another _plugin_, which is less likely.